### ISSUE TITLE
Obfuscated server selection.

### ISSUE THREAD

**Issue by adamczykm** [jotyGill/openpyn-nordvpn#212]
Currently there's no option to connect to the obfuscated servers.
It'd be very useful as some networks blocks unobfuscated servers.

**Comment by larry77** [comment]
Any progress on this? I see that nordvpn has released a linux app, but I would like to stick to this wonderful open source project!

**Comment by Angel418** [comment]
Yep, would love to have this functionality. Recent NordVPN app updates have broken it on my system, and my network requires obfuscation (bad, evil, malicious ISP) to function properly.

**Comment by larry77** [comment]
Once again, any news on this? I understand this may not be crucial,
but it is a desirable functionality.

PS: of course no criticism meant about the tools kindly offered by
jotyGill. It is a real gem to have an open-source tool to connect to
nordvpn as opposed to an opaque proprietary app.

**Comment by 1951FDG** [comment]
This functionality is already in place,

`--anti-ddos` Only look for servers with "Obfuscated" support

**Comment by larry77** [comment]
Apologies and my mistake then.  Keep up the good work.

On 10 July 2019 12:19:48 CEST, Derrick <notifications@github.com> wrote:
>This functionality is already in place,
>
>`--anti-ddos` Only look for servers with "Obfuscated" support
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/jotyGill/openpyn-nordvpn/issues/212#issuecomment-510003442

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by mjsp** [comment]
Unfortunately I don't find any servers with the --anti-ddos option, I think this is because the obfuscated servers are instead provided at a separate URL now:
https://downloads.nordcdn.com/configs/archives/servers/ovpn_xor.zip

But given it requires a patched version of openvpn to handle these scramble / obfuscated options for it to work and that the patch is not supported by the authors of openvpn, it's maybe not desirable to have it download the xor ones for everyone.

For me, it would be helpful just for the URL to be configurable so that I can force it to download the xor ones instead.
